lahja/asyncio/endpoint.py

```diff
@@ -117,6 +117,13 @@ def __init__(self, reader: StreamReader, writer: StreamWriter) -> None:
                 await self.writer.drain()
         except (BrokenPipeError, ConnectionResetError):
             raise RemoteDisconnected()
+        except RuntimeError as err:
+            # We don't do a pre-check for a closed writer since this is a performance critical
+            # path. We also don't want to swallow runtime errors unrelated to closed handlers.
+            if "handler is closed" in str(err):
+                self.logger.warning("Failed to send %s. Handler closed.", message)
+                return
```

Comment:
what about doing `raise RemoteDisconnected from err` here?  Makes it catchable using the same mechanism above.

Reply:
Ah sure! :+1: